### PR TITLE
🔖 Version 4.5.4

### PR DESCRIPTION
**Pull request by phatblat**
Our automates release failed to publish an update to this package because of branch protection on `main`. I've manually updated the package manifest in this PR and will tag a new release after approval of this PR.

```
#### Generating Package.swift ####
// swift-tools-version:5.5
import PackageDescription
let package = Package(
    name: "Ditto",
    platforms: [ .iOS(.v11), .macOS(.v11) ],
    products: [
        .library(
            name: "DittoSwift",
            targets: ["DittoSwiftWrapper"]),
        .library(
            name: "DittoObjC",
            targets: ["DittoObjC"]),
    ],
    targets: [
        .target(
            name: "DittoSwiftWrapper",
            dependencies: [
                .target(name: "DittoSwift"),
                .target(name: "DittoObjC"),
            ]
        ),
        .binaryTarget(
            name: "DittoSwift",
            url: "https://software.ditto.live/cocoa/DittoSwift/4.5.4/dist/DittoSwift.xcframework.zip",
            checksum: "7463a30591c9db2ca6ae37e74c4d2d2a5b19f812b29a1ce52fc54e90f5297442"
        ),
        .binaryTarget(
            name: "DittoObjC",
            url: "https://software.ditto.live/cocoa/DittoObjC/4.5.4/dist/DittoObjC.xcframework.zip",
            checksum: "d34100e7f755ee61161a3f3d048113ffd8289ba41f552807c61c60853064fecc"
        ),
    ]
)
#### Pushing to @dittolive/DittoSwiftPackage ####
+ cd /usr/local/var/buildkite-agent/builds/co-mac1/dittolive-incorporated/ditto/cocoa/DittoSwiftPackage
+ git add .
+ git commit -m 'Release 4.5.4'
[main f38a916] Release 4.5.4
 1 file changed, 4 insertions(+), 4 deletions(-)
+ git tag 4.5.4
+ git push git@github.com:getditto/DittoSwiftPackage.git refs/heads/main:refs/heads/main refs/tags/4.5.4:refs/tags/4.5.4
Enumerating objects: 5, done.
Counting objects: 100% (5/5), done.
Delta compression using up to 36 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 414 bytes | 414.00 KiB/s, done.
Total 3 (delta 2), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/main.
remote: error: Changes must be made through a pull request.
To github.com:getditto/DittoSwiftPackage.git
 * [new tag]         4.5.4 -> 4.5.4
 ! [remote rejected] main -> main (protected branch hook declined)
error: failed to push some refs to 'github.com:getditto/DittoSwiftPackage.git'
```

https://buildkite.com/dittolive-incorporated/ditto/builds/53899#018dca08-41e1-4f85-b352-b2b2b43f8b09/6-3413

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -23,13 +23,13 @@ let package = Package(
         ),
         .binaryTarget(
             name: "DittoSwift",
-            url: "https://software.ditto.live/cocoa/DittoSwift/4.5.3/dist/DittoSwift.xcframework.zip",
-            checksum: "2b7f4f8e070e38c9dc4674e98df78a76fb7e51999a1b2d6fc2f0a04d724ad567"
+            url: "https://software.ditto.live/cocoa/DittoSwift/4.5.4/dist/DittoSwift.xcframework.zip",
+            checksum: "7463a30591c9db2ca6ae37e74c4d2d2a5b19f812b29a1ce52fc54e90f5297442"
```
Comment (radazzouz) [inline Package.swift:27]: The checksum I got after running `swift package compute-checksum DittoSwift.xcframework.zip` locally is different

```suggestion
            checksum: "d5ab849b194d6d4e0f66e6a3211d0cfdc94921469126484d9361b530ee5a5624"
```

Comment (phatblat) [reply Package.swift]: Thank you so much for catching this!

**Comment by phatblat** [comment]
I've been working around this problem by disabling branch protection manually but I forgot before this SDK release.

**Review by radazzouz — APPROVED** [review]
I tested locally and it works.

Screenshot 2024-02-21 at 5 48 52 PM

**Review by magicwave — APPROVED** [review]

**Pull request merged**